docling_serve/datamodel/responses.py

```diff
@@ -35,6 +35,10 @@ class ConvertDocumentResponse(BaseModel):
     timings: dict[str, ProfilingItem] = {}
 
 
+class PresignedUrlConvertDocumentResponse(BaseModel):
+    url: str
+
```

Comment:
let's keep `status` and `processing_time` from the other type of response (`ConvertDocumentResponse`)